lib/berlim/company_accounts/company.ex

```diff
@@ -0,0 +1,37 @@
+defmodule Berlim.CompanyAccounts.Company do
+  @moduledoc false
+
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  import Comeonin.Bcrypt, only: [hashpwsalt: 1]
```

Comment:
to atualizando uns projetos e saiu uns updates aqui, com break changes, da uma olhada no https://github.com/riverrun/comeonin/blob/master/UPGRADE_v5.md